docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-shipment-feature-integration.md

```diff
@@ -72,7 +72,7 @@ class ShipmentConfig extends SprykerShipmentConfig
 
 {% info_block warningBox "Verification" %}
 
-Make sure that `ShipmentService::groupItemsByShipment()` groups items by shipment using the merchant reference.
+Place an order and check that items are grouped by merchant shipment in backoffice.
```

Comment:
looks like a wrong verification, it is already used in another IG

Reply:
I've mentioned this verification twice. Plugin that groups items in backoffice is integrated here, but I have no idea, how to check it without placing an order. And order can't be placed without `Shipment + Cart` integration.